### ISSUE TITLE
COMMUNITY: Regular Python-graphblas community conference calls (Open to all)

### ISSUE THREAD

**Issue by eriknw** [python-graphblas/python-graphblas#247]
## tl;dr:
- Weekly calls at 2:30pm GMT / 10:30am ET / 7:30am PT for about 1 hour
- Zoom link: https://anaconda.zoom.us/j/97267234756?pwd=Q00wSjE2dDV6NmF3eFBZSFBtVDFSQT09
    - Meeting ID: 972 6723 4756
    - Passcode: 584468
- Google calendar: TODO
- ical format (ics): TODO
- Rolling agenda/minutes: TODO
- If you are unable to join (i.e., broken link, or time doesn't work for you), please comment below and we'll try to accomodate.

## Description
Python-graphblas community meetings occur weekly.  This is open to anyone, and new users or contributors looking for guidance are most welcome to join us.  We typically get together to discuss issues, design, features, processes, or anything important for the progress of `python-graphblas`, `graphblas-algorithms`, or`dask-graphblas`.  We also like to show things off, so if you have something to show off, please do!

**Comment by jim22k** [comment]
For anyone who has attempted to join the meeting recently at 2:00pm GMT, we are aware of the challenge of timezones for global meetings. The actual meeting time is based on daylight savings in the US, so it will be either 2:00pm GMT in the summer or 3:00pm GMT in the winter. Sorry for the confusion.
